python/extensions/pip.bzl

```diff
@@ -247,14 +263,27 @@ def _pip_parse_ext_attrs():
         "hub_name": attr.string(
             mandatory = True,
             doc = """
-The unique hub name.  Mulitple pip.parse calls that contain the same hub name, 
-create spokes for specific Python versions.                                
+The repo name pip dependencies will be made available within.
+
+This name must be unique between modules, but should be the same within
```

Comment:
You can have more than one hub name in a Module.  I could see it used in larger projects that have different requirements files.  So "should be the same", can we do something like "typically the same"?

Reply:
Ah yes, good catch. Updated the text to clarify that better.